Source/Atomic/UI/UI.cpp

```diff
@@ -535,11 +539,52 @@ void UI::HandleUpdate(StringHash eventType, VariantMap& eventData)
         exitRequested_ = false;
         return;
     }
+    
+    mouseStayedTime += eventData[Update::P_TIMESTEP].GetFloat();
+
+    if (mouseStayedTime >= 0.5f)
+    {
+        UIWidget* hoveredWidget = GetHoveredWidget();
+        if (hoveredWidget && !tooltip_ && (hoveredWidget->GetShorten() || hoveredWidget->GetTooltip().Length() > 0))
+        {
+            tooltip_ = new UIPopupWindow(context_, hoveredWidget, "tooltip");
```

Comment:
Ok, good to know, are we sure we want to use TBPopupWindow?  I guess I haven't looked at how this appears in the editor, @LaraEngebretson will kick off a local build and check it out :)